### ISSUE TITLE
BLM overlay is too large on mobile

### ISSUE THREAD

**Issue by jasikpark** [ml5js/ml5-website-archive#175]
![2A31D1B6-3118-4AE6-9CFD-0131F443F38C](https://user-images.githubusercontent.com/10626596/110249728-24ad4580-7f3d-11eb-9b5c-cd604145ce7f.png)

The overlay goes past the search bar and the bottom chrome, I assume there’s a close button up there?

It’s probably a 100vh bug - mobile browsers (webkit) aren’t 100% consistent about it. 

https://css-tricks.com/css-fix-for-100vh-in-mobile-webkit/

**Comment by bomanimc** [comment]
Good catch @jasikpark! This is one of a few issues on our mobile site that I'd love for us to address!
